ovos_utils/json_helper.py

```diff
@@ -26,7 +26,7 @@ def merge_dict(base, delta, merge_lists=False, skip_empty=False,
         else:
             if new_only and k in base:
                 continue
-            if skip_empty and not d:
+            if skip_empty and not d and d is not False:
```

Comment:
Will this also skip `0`? Maybe it should just be `d is None`

Reply:
i think it should also skip empty lists and empty strings, but you are right that 0 should not be skipped

perhaps a check for isinstance ?

Reply:
`all((i == None for i in {}))` will return correctly on iterables.